lib/paper_trail/serializers/yaml.rb

```diff
@@ -23,7 +23,7 @@ def where_object_condition(arel_field, field, value)
       # in the serialized object_changes
       def where_object_changes_condition(arel_field, field, value)
         # Need to check first (before) and secondary (after) fields
-        if defined?(::YAML::ENGINE) && ::YAML::ENGINE.yamler == 'psych'
+        if defined?(::YAML::ENGINE) && ::YAML::ENGINE.yamler == 'psych' || ::YAML == Psych
```

Comment:
Can you qualify Psych with `::Psych`? I think that would make the Travis errors stop failing.